tools/integration_tests/managed_folders/admin_permissions_test.go

```diff
@@ -207,13 +203,27 @@ func TestManagedFolders_FolderAdminPermission(t *testing.T) {
 
 	for i := 0; i < len(permissions); i++ {
 		log.Printf("Running tests with flags, bucket have %s permission and managed folder have %s permissions: %s", permissions[i][0], permissions[i][1], flags)
+		bucket, testDir = setup.GetBucketAndObjectBasedOnTypeOfMount(TestDirForManagedFolderTest)
 		ts.bucketPermission = permissions[i][0]
 		if ts.bucketPermission == ViewPermission {
 			creds_tests.RevokePermission(serviceAccount, AdminPermission, setup.TestBucket())
 			creds_tests.ApplyPermissionToServiceAccount(serviceAccount, ViewPermission)
 			defer creds_tests.RevokePermission(serviceAccount, ViewPermission, setup.TestBucket())
 		}
 		ts.managedFolderPermission = permissions[i][1]
+		if ts.managedFolderPermission != "nil" {
```

Comment:
is ts.bucketPermission and ts.managedFolderPermission being used anywhere else? If not, we can probably remove it from test structure?

Reply:
`ts.bucketPermission` is getting used in test cases.  ts.managedFolderPermission  removed from test structure.